README.rst

```diff
@@ -0,0 +1,81 @@
+===========================
+ OpenSlides RTC Plugin
```

Comment:
You should use "openslides-conversations" also as name/path of the plugin - instead of "openslides-rtc".